src/Knp/DictionaryBundle/DependencyInjection/Configuration.php

```diff
@@ -19,37 +19,38 @@ public function getConfigTreeBuilder()
             ->root('knp_dictionary')
             ->children()
             ->arrayNode('dictionaries')
-                ->useAttributeAsKey('name')
-                    ->prototype('array')
-                        ->beforeNormalization()
-                            ->always()
-                            ->then(function ($values) {
-                                if (false === array_key_exists('type', $values)) {
-                                    if (false === array_key_exists('content', $values)) {
-                                        return ['type' => 'value', 'content' => $values];
-                                    }
+            ->useAttributeAsKey('name')
```

Comment:
I know you're kinda strict about cs fixing, but for this specific file, the indentation helps a lot to make things more readable. So I think we should ignore this class from cs fixing :)